src/app/teacher/group-detail/group-detail.component.ts

```diff
@@ -7,9 +8,36 @@ import { Component, OnInit } from '@angular/core';
 })
 export class GroupDetailComponent implements OnInit {
 
-  constructor() { }
+  id: Number;
+  items: Object;
+  selected: Object;
+
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router
+  ) {
+  }
 
   ngOnInit(): void {
+    this.id = this.route.snapshot.params["id"];
+    this.items = [
+      { name: "Budget", id: 0,  subItems: [{ name: "L1: Income 1 attempt 2/10" }, { name: "L2: Fix Expenses 2 attempts 9/10" }, { name: "L3: Variable Expenses 3 attempts 10/10 " }] },
```

Comment:
Should be in a JSON file in `/assets`. Keeps the code cleaner.

Reply:
Don't have time to import, can do it tomorrow if we have time.